index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Code Reviewer Trials</title>
+  </head>
+  <body>
+    <header>
+      <p>Hello Microverse</p>
+    </header>
```

Comment:
- [ ] Kindly update this `header` tag in an `h1` tag, as this is a project requirement.
- [ ] Also consider putting the content of the header directly in the `h1` tag, not in the `p` tag